modules/enable-openshift-monitoring-central-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-operator_{context}"]
+= Monitoring Central services using the {product-title-short} Operator
+
+[role="_abstract"]
+You can monitor Central services, Central and Scanner, by changing the configuration of the `Central` custom resource.
+
+.Procedure
+. On the {ocp} web console, go to the *Operators* → *Installed Operators* page.
+. Select the {product-title} Operator from the list of installed Operators.
+. Click on the *Central* tab.
+. From the list of Central instances, click on a Central instance for which you want to enable monitoring for.
+. Click on the *YAML* tab and update the YAML configuration:
+* For monitoring Central and Scanner, enable the `monitoring.openshift.enabled` configuration option for the `Central` custom resource.
++
+[source,yaml]
+----
+apiVersion: platform.stackrox.io/v1alpha1
+kind: Central
+...
+spec:
+  monitoring:
+    openshift:
+      enabled: true
+...
+----
```

Comment:
Something is wrong here - I'm not sure what - but if you look at the preview, you can see the problem. Somehow there is a bunch of other stuff pulled into the example....

![Screenshot 2023-08-08 at 1 54 14 PM](https://github.com/openshift/openshift-docs/assets/96192119/06488679-ada5-4fa7-b93e-81af9afdd8a0)

Reply:
But the code looks OK, so maybe the preview didn't update?

Reply:
I have problem with previews are not running for me :( I run it locally with 
- https://github.com/openshift/openshift-docs/pull/63201
and added missing `---` in https://github.com/openshift/openshift-docs/pull/63050/commits/eae9d55e0f3cf007d395d1b35a51ae90977e6974